index.html

```diff
@@ -6582,92 +6595,136 @@ <h2 id="instantiation-of-AudioWorkletNode-and-AudioWorkletProcessor">
+            by <var>nodeName</var> from <a>node name to parameter descriptor
+            map</a>:
               <ol>
-                <li>Let <var>audioParam</var> be a new <a>AudioParam</a>
-                instance with:
-                  <ul>
-                    <li>
-                      <var>paramName</var> being the result <a href=
-                      "https://tc39.github.io/ecma262/#sec-get-o-p">Get</a>(O=<var>descriptor</var>,
-                      P="name").
-                    </li>
-                    <li>
-                      <var>defaultValue</var> being the result <a href=
-                      "https://tc39.github.io/ecma262/#sec-get-o-p">Get</a>(O=<var>descriptor</var>,
-                      P="defaultValue").
+                <li>Let <var>audioParamMap</var> be a new <a>AudioParamMap</a>
+                object.
+                </li>
+                <li>For each <var>descriptor</var> of
+                <var>parameterDescriptors</var>:
+                  <ol>
+                    <li>Let <var>paramName</var> be the value of
+                    <var>descriptor</var>'s <a href=
+                    "#widl-AudioParamDescriptor-name">name</a>.
                     </li>
-                    <li>
-                      <var>minValue</var> being the result <a href=
-                      "https://tc39.github.io/ecma262/#sec-get-o-p">Get</a>(O=<var>descriptor</var>,
-                      P="minValue").
+                    <li>Let <var>audioParam</var> be a new <a>AudioParam</a>
+                    instance.
                     </li>
-                    <li>
-                      <var>maxValue</var> being the result <a href=
-                      "https://tc39.github.io/ecma262/#sec-get-o-p">Get</a>(O=<var>descriptor</var>,
-                      P="maxValue").
+                    <li>Append (<var>paramName</var>, <var>audioParam</var>) to
+                    <var>audioParamMap</var>'s entries.
                     </li>
-                  </ul>
+                  </ol>
                 </li>
-                <li>If <var>options</var> contains a dictionary member with
-                name <var>paramName</var> whose value is of type
-                <code>Number</code>, perform <a href=
-                "https://tc39.github.io/ecma262/#sec-set-o-p-v-throw">Set</a>(<var>audioParam</var>,
-                "value", <a href="https://tc39.github.io/ecma262/#sec-get-o-p">
-                  Get</a>(O=<var>options</var>, P=<var>paramName</var>),
-                  false).
+                <li>For each <var>paramNameInOption</var> → <var>value</var> of
+                <var>options</var>:
+                  <ol>
+                    <li>If there exists an entry with name member equal to
+                    <var>paramNameInOption</var> inside
+                    <var>audioParamMap</var>, set that <a>AudioParam</a>'s
+                    value to <var>value</var>.
+                    </li>
```

Comment:
Maybe describe what happens if the entry doesn't exist? (In code, it's obvious, but with text and lists, it's not so clear.)